app/controllers/catalog_controller.rb

```diff
@@ -182,9 +182,15 @@ def show
   end
 
   def dc
+    respond_to do |format|
+      format.html { render layout: request.xhr? }
```

Comment:
In some methods added we have the logic as `!request.xhr?` but here its `request.xhr?` why would we want different logic in these similar stanzas?

Reply:
Nope, that's a mistake.